perfkitbenchmarker/providers/aws/aws_network.py

```diff
@@ -64,6 +64,28 @@ def __init__(self):
     self.firewall_set = set()
     self._lock = threading.Lock()
 
+  def AllowIcmp(self, vm):
+    """Opens the ICMP protocol on the firewall.
+    Args:
+      vm: The BaseVirtualMachine object to open the ICMP protocol for.
+    """
+    print("ALLOWING ICMP")
```

Comment:
Why do you print status here and not in gce_network.py?

Reply:
Forgot to remove a debug statement. It is gone now